boards/arm/nrf9160_pca10090/nrf9160_pca10090.yaml

```diff
@@ -0,0 +1,9 @@
+identifier: nrf9160_pca10090
+name: nRF9160-PCA10090
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+ram: 64

```

Comment:
Same comment as for the Non-Secure board; I added what is actually available by default with this board definition.